.github/workflows/ci.yml

```diff
@@ -4,9 +4,9 @@ name: CI
 # events but only for the master branch
 on:
   push:
-    branches: [ main, develop, ux-redesign ]
+    branches: [ main, feature/high-prevalence ]
```

Comment:
Do you wanna merge this PR into the new branch your proposing? Or into main (as it's set to right now)?

Reply:
Mmph, I mean to land this in main because it doesn't change behaviors. This ci change can be excluded, though.